locust_plugins/jmeter_listener.py

```diff
@@ -136,11 +144,22 @@ def add_result(self, success, _request_type, name, response_time, response_lengt
             connect,
         ]
         self.csv_results.append(self.field_delimiter.join(row))
-        if len(self.csv_results) >= self.flush_size:
+        if len(self.csv_results) >= self.flush_size and not self.is_worker_runner:
             self._flush_to_log()
 
     def _request_success(self, request_type, name, response_time, response_length, **kw):
         self.add_result("true", request_type, name, response_time, response_length, "", **kw)
 
     def _request_failure(self, request_type, name, response_time, response_length, exception, **kw):
         self.add_result("false", request_type, name, response_time, response_length, str(exception), **kw)
+
+    def _report_to_master(self, client_id, data):
+        client_id = client_id
```

Comment:
This is weird :)

Reply:
happy to take suggestions...

Reply:
I just meant: why are you assigning a variable to itself?

Reply:
...which is how I had it to start with. I'd appreciate some guidance on how this method should look. Thanks

Reply:
Just name the variable _client_id if you all you want is to indicate that it is unused (and remove the client_id = client_id line)

Reply:
I see it's a keyword argument so I just added it to _kwargs